Bump Snapshot Version during release fails

## Describe the bug

During Release workflow, Bump Snapshot Version step fails due to commit disabled on main branch.

Previous run where it failed
https://github.com/factory-x-contributions/factoryx-edc/actions/runs/18287615711/job/52069479370

### To Reproduce
Run a release workflow


### Expected behavior
Commit should be allowed on main and release workflow should not fail.

### Screenshots/Error Messages
```
[main 512cb61] Introduce new snapshot version 0.1.4-SNAPSHOT
 1 file changed, 1 insertion(+), 1 deletion(-)
remote: error: GH013: Repository rule violations found for refs/heads/main.        
remote: Review all repository rules at https://github.com/factory-x-contributions/factoryx-edc/rules?ref=refs%2Fheads%2Fmain        
remote: 
remote: - Changes must be made through a pull request.        
remote: 
remote: - Commits must have verified signatures.        
remote:   Found 1 violation:        
remote: 
remote:   512cb617b12614eb1b1395b1f155542dbeb8784e        
remote: 
To https://github.com/factory-x-contributions/factoryx-edc
 ! [remote rejected] main -> main (push declined due to repository rule violations)
error: failed to push some refs to 'https://github.com/factory-x-contributions/factoryx-edc'
Error: Process completed with exit code 1.
```

## Context Information

_Add any other context or information about the problem here._

- Used version: [e.g. Commit Hash]
- OS: [e.g. Mac OS (M1), Windows, Linux]
- Docker Version: [e.g. 20.10.12]
- `java --version`:

## Possible Implementation

_You already know the root cause of the erroneous state and how to fix it? Feel free to share your thoughts._

--- a/.github/workflows/bump-version.yaml
+++ b/.github/workflows/bump-version.yaml
@@ -32,6 +32,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v5
+        with:
+          token: '${{ secrets.GH_PAT }}'
       - name: Set new snapshot version
         shell: bash
         run: |

